charts/argo-cd/README.md

```diff
@@ -278,6 +278,11 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 6.8.0
+
+Added support for IPv4/IPv6 dual-stack networking.
+Please refer to [the docs](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) for more information.
+
```

Comment:
This PR doesn't have breaking change, so I think we don't need changelog. 🙋

Reply:
You only add changelog for breaking change ? In 6.1.0 you added a line when you supported a new option :sweat_smile:. I can remove it if you want :)

Reply:
>In 6.1.0 you added a line when you supported a new option 😅

Since it affected to existing attributes. :)